Jenkinsfile.trigger

```diff
@@ -102,69 +102,100 @@ if (breakEarly) { return } // thanks Jenkins...
 
 // new data to be added to the past-jobs.json
 // { lastTime: unixTimestamp, url: "" }
-def buildCompletionData = [:]
-
-for (buildObj in queue) {
-	stage(buildObj.identifier) {
-		//def json = writeJSON(json: buildObj, returnText: true)
-		//echo(json) // for debugging/data purposes
-
-		// "catchError" to set "stageResult" :(
-		catchError(message: 'Build of "' + buildObj.identifier + '" failed', buildResult: 'UNSTABLE', stageResult: 'FAILURE') {
-			if (buildObj.gha_payload) {
-				node {
-					withEnv([
-						'payload=' + buildObj.gha_payload,
-					]) {
-						withCredentials([
-							string(
-								variable: 'GH_TOKEN',
-								credentialsId: 'github-access-token-docker-library-bot-meta',
-							),
+buildCompletionData = [:]
```

Comment:
> Did you forget the `def` keyword? WorkflowScript seems to be setting a field named buildCompletionData (to a value of type LinkedHashMap) which could lead to memory leaks or other issues.

This is a warning that I saw in Jenkins and just wanted to document our choice here.  We needed global scope (no `def`) so that the `waitQueueClosure` function could access this variable. We chose no `def` over `@Field` because `@Field \n def` is more confusing than a simple global variable.

https://stackoverflow.com/a/50573082